lib/modules/apostrophe-pages/index.js

```diff
@@ -291,6 +291,76 @@ module.exports = {
       ], callback);
     };
 
+    // Task to add a page from apostrophe
+    // command line
+
+    self.apos.tasks.add('apostrophe-pages', 'add',
+      'Usage: node app apostrophe-pages:add title /slug type /parentslug\n\n' +
+      'This adds a new page .\n',
+      function(apos, argv, callback) {
+        return self.addFromTask(callback);
+      }
+    );
+
+    self.addFromTask = function(callback) {
+      var argv = self.apos.argv;
+      if(argv._.length !== 5 ){
+        return callback('Incorrect number of arguments.');
+      }
+
+      var title = argv._[1];
+      var slug = argv._[2];
+      var type = argv._[3];
+      var parentslug = argv._[4];
+      var req = self.apos.tasks.getReq();
+
+      var page = {
+          editUsersIds: [],
+          loginRequired: '',
+          orphan: '0',
+          published: '1',
+          slug: slug,
+          tags: [],
+          title: title,
+          type: type
+        };
+
+      
+      var parentPage;
+      var safePage;
+      return async.series({
+        findParent: function(cb) {
+          return self.find(req, { slug: parentslug }).permission('publish-apostrophe-page').toObject(function(err, _parentPage) {
+            if (err) {
+              return cb(err);
+            }
+            if (!_parentPage) {
+              return cb('Parent page not found');
+            }
+            parentPage = _parentPage;
+            safePage = self.newChild(parentPage);
+            return cb(null);
+          });
+        },
+        convert: function(cb) {
+          var manager = self.apos.docs.getManager(self.apos.launder.string(page.type));
+          if (!manager) {
+            return cb('Template not found');
+          }
+          var schema = manager.allowedSchema(req);
+          return self.apos.schemas.convert(req, schema, 'form', page, safePage, cb);
```

Comment:
This would be a good place to loop over the schema and look for matching arguments in argv and push those into `page`.